mac/Keyman4Mac/Keyman4Mac/AppDelegate.m

```diff
@@ -135,80 +133,17 @@ CGEventRef eventTapFunction(CGEventTapProxy proxy, CGEventType type, CGEventRef
                 CFRelease(kEventDown);
```

Comment:
Why do we still have `contextBuffer` on line 132 above?

Reply:
deleted